src/overlays/actors/ovl_En_M_Thunder/z_en_m_thunder.c

```diff
@@ -29,39 +39,533 @@ const ActorInit En_M_Thunder_InitVars = {
     (ActorFunc)EnMThunder_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_808B7120 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_PLAYER, AC_NONE, OC1_NONE, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK2, { 0x01000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NONE, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_PLAYER,
+        AC_NONE,
+        OC1_NONE,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK2,
+        { 0x01000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NONE,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 200, 200, 0, { 0, 0, 0 } },
 };
 
-#endif
+static u8 sDamages[] = {
+    /* Regular    */ 1, 2, 3, 4,
+    /* Great Spin */ 1, 2, 3, 4,
+};
+
+static u16 sSounds[] = {
+    NA_SE_IT_ROLLING_CUT_LV2,
+    NA_SE_IT_ROLLING_CUT_LV1,
+    NA_SE_IT_ROLLING_CUT_LV2,
+    NA_SE_IT_ROLLING_CUT_LV1,
+};
```

Comment:
```suggestion
static u16 sSpinAttackSfxIds[] = {
    NA_SE_IT_ROLLING_CUT_LV2,
    NA_SE_IT_ROLLING_CUT_LV1,
    NA_SE_IT_ROLLING_CUT_LV2,
    NA_SE_IT_ROLLING_CUT_LV1,
};
```
As an idea, it should definitely have `SfxIds` in there is some capacity

Reply:
Felt like `sChargingSfxIds` would be best. Also added comments with the enum value each entry corresponds with.